README.md

```diff
@@ -129,9 +129,36 @@ databases:
 
 ### Camera configuration
 
-The camera component tries to guess everything from the rrd file. But you can always pass new arguments in *args*.
+The camera component tries to guess everything from the rrd file. Alternatively you can always pass new arguments in `args`.
+
+#### `args`
+
+For basic rrd graph, you do not need to make any own configuration in `args`.
+In this case rrd graph will render a line for each datasource of `rrdfile` automatically.
+
+For advanced graph rendering you can use `args` for definitions of parameters CDEF, VDEF, LINE1, ARRAY, etc.
+If you will use own `args` configuration there is automatically generated DEF for each RRA of each datasource.
+
+**Example:**
+
+Rrd file `example1.rrd` has configuration:
+
+    rrdtool create example1.rrd \
+    --step '900' \
+    'DS:temperature:GAUGE:1800:U:U' \
+    'RRA:AVERAGE:0.5:4:48' \
+    'RRA:AVERAGE:0.5:96:365'     
+
+For this `example1.rrd` there are following DEF variables, which you can use in `args`.
+
+- `Temperature`
+- `Temperature_AVERAGE_4`
+- `Temperature_AVERAGE_96`
```

Comment:
Considering the naming convention of primary RRA. Maybe is should be same as others. Something like `Temperature_AVERAGE` or `Temperature_AVERAGE_1` (instead of current `Temperature`).

... But in this case it will be breaking change. :-(

Reply:
:) OK. So I am offering to rename primary DEF to `{Capitalize_DS}_{cf}` (example `Temperature_AVERAGE`). Using of `_1` looks unclear and redundant. Do you agree with me?

... Second question is, why to capitalize all DS in DEF? (It was in original code.) Is it recommended usage? I am asking, because I was confused of this at first usage (Why name is not as original datasource. I mean without capitalizing.)

Reply:
I was just keeping up with the old MRTG tradition :)

I'm fine with the change, but lets open a separate PR just to keep matters separated.